README.md

```diff
@@ -57,6 +57,11 @@ Zulip](https://lowrisc.zulipchat.com/#narrow/stream/198227-ibex)!
 Unless otherwise noted, everything in this repository is covered by the Apache
 License, Version 2.0 (see LICENSE for full text).
 
+## Credits
+
+Many people have contributed to Ibex through the years. Please have a look at
+the [credits file](CREDITS.md) for more information.
```

Comment:
`the [credits file](CREDITS.md) and the commit history for more information.`

Reply:
updated.